### PR TITLE
Add lists of inactive Reps > 12months, > 8 weeks to administartive da…

### PR DESCRIPTION
**Pull request by MichaelKohler**
…shboard

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@irvin do you think something like 6 months or 3 months would be helpful as well?

**Comment by MichaelKohler** [comment]
@ioana-chiorean also same question for you, what period would help you?

**Comment by Mte90** [comment]
does have sense to have 8 weeks and 3 months? after we are talking about 2 and 3 months.

**Comment by MichaelKohler** [comment]
actually even better, now it's a list of 6 months and 12 months of inactivity, as well as *active* Reps in /stats

**Comment by Mte90** [comment]
what is the `/stats` page?

File: remo/dashboard/templates/dashboard_reps.jinja [hunk 1]
```diff
@@ -1030,6 +1030,114 @@
       </div>
     </div>
     {% endif %}
+    {% if reps_inactive12 is defined %}
+    <div class="section">
```
Comment (akatsoulas) [inline remo/dashboard/templates/dashboard_reps.jinja]: Indent 2 spaces in please :)

File: remo/dashboard/views.py [hunk 2]
```diff
@@ -178,6 +178,12 @@ def dashboard(request):
                                           reps_without_mentor_q)
         order_by = '-userprofile__date_joined_program'
         args['reps_without_mentors'] = reps_without_mentor.order_by(order_by)
+        q_active_12 = Q(
```
Comment (akatsoulas) [inline remo/dashboard/views.py]: s/q_active_12/q_active_12_months

File: remo/dashboard/views.py [hunk 3]
```diff
@@ -178,6 +178,12 @@ def dashboard(request):
...
+            ng_reports__report_date__range=[get_date(weeks=-52), get_date(weeks=0)])
+        args['reps_inactive12'] = reps.filter(~q_active_12).distinct()
```
Comment (akatsoulas) [inline remo/dashboard/views.py]: nit: let's use an `_` here too for consistency. Also pls append `months`

Comment (akatsoulas) [reply remo/dashboard/views.py]: This has more occurrences.

**Comment by Mte90** [comment]
What period is helpful for the program to show the reps inactive.

**Comment by irvin** [comment]
I don't see the list at https://reps.mozilla.org/dashboard/stats/ , is it up?

**Comment by Mte90** [comment]
no they didn't deployed yet any changes...

**Comment by irvin** [comment]
okay I will need to wait for this up to start working on https://github.com/mozilla/Reps/issues/375

**Pull request merged**